networkx/algorithms/link_prediction.py

```diff
@@ -277,17 +277,27 @@ def common_neighbor_centrality(G, ebunch=None, alpha=0.8):
            https://doi.org/10.1038/s41598-019-57304-y
     """
 
-    spl = dict(nx.shortest_path_length(G))
-    inf = float("inf")
+    # Avoid computing shortest path lengths for entire graph unless needed
```

Comment:
```suggestion
    # When alpha == 1, the CCPA score simplifies to the number of common
    # neighbors
```
Just a wording suggestion to avoid the forward reference to the path lengths.

Reply:
Good call, I've updated the comment 👍